### PR TITLE
Add GPU docs

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: components/plans.tsx [hunk 1]
```diff
@@ -130,19 +130,67 @@ const PLANS_LIST = [
...
+  },
+  {
+    title: "Nvidia A4000",
+    amount: "349.49",
+    dailyAmount: "11.65",
+    specs: [16, 8],
+  },
+  {
+    title: "Nvidia A100",
+    amount: "548.25",
```
Comment (rekpero) [inline components/plans.tsx:154]: fix the pricing. check the current price

**Review by rekpero — CHANGES_REQUESTED** [review]

File: components/plans.tsx (hunk 1 above)
Comment (rekpero) [reply components/plans.tsx:154]: check other plans as well

Comment (aayushmahapatra) [reply components/plans.tsx:166]: Done. A100 is the same. Updated the others.

File: components/plans.tsx [hunk 2]
```diff
@@ -130,19 +130,67 @@ const PLANS_LIST = [
   },
 ];
 
-const Plans = () => {
+const GPU_PLANS_LIST = [
+  {
+    title: "Nvidia 3080",
+    amount: "305.53",
+    dailyAmount: "10.18",
```
Comment (rekpero) [inline components/plans.tsx:137]: show hourly rate instead of daily

**Review by rekpero — CHANGES_REQUESTED** [review]

File: components/plans-card.tsx [hunk 3]
```diff
@@ -1,23 +1,45 @@
...
+          )}
+          <h3 className={styles.heading}>{title}</h3>
+        </div>
         <p className={styles.amount}>
           <span className={styles.amountValue}>${amount}</span>{" "}
           <span className={styles.paragraph}>Avg monthly</span>
         </p>
-        <p className={styles.dailyAmount}>${dailyAmount} avg daily cost</p>
+        {isGPU ? (
+          <p className={styles.dailyAmount}>${hourlyAmount} avg hourly cost</p>
```
Comment (rekpero) [inline components/plans-card.tsx:39]: make hourly primary for both gpu and CPU 
also convert the cpus to hourly rate

**Review by rekpero — CHANGES_REQUESTED** [review]

File: components/plans.tsx [hunk 4]
```diff
@@ -130,19 +138,75 @@ const PLANS_LIST = [
...
+    specs: [16, 8],
+  },
+  {
+    title: "Nvidia A100",
+    amount: "548.25",
+    dailyAmount: "18.28",
+    hourlyAmount: "0.76",
+    specs: [32, 16],
+  },
+  {
```
Comment (rekpero) [inline components/plans.tsx:184]: remove v100 from the list

**Review by rekpero — CHANGES_REQUESTED** [review]

**Review by rekpero — APPROVED** [review]

**Pull request merged**